jobs/build/update-base-images/Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+node {
+    checkout scm
+    def commonlib = load("pipeline-scripts/commonlib.groovy")
+    def buildlib = load("pipeline-scripts/buildlib.groovy")
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '100')
+                ),
+                [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'BASE_IMAGE_TAGS',
+                        description: 'list of openshift cve base images.',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: commonlib.ocpBaseImages.join(',')
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-team-art@redhat.com',
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+            disableConcurrentBuilds()
+        ]
+    )
+
+    commonlib.checkMock()
+    buildlib.initialize()
+    def imageName = params.BASE_IMAGE_TAGS.split(',')
+
+
+    currentBuild.displayName = "#${currentBuild.number} Update base images"
+    currentBuild.description = ""
+    try {
+        imageName.each { image ->
+            sh "./build.sh ${image}"
```

Comment:
Capture the stdout/stderr to artifacts with `commonlib.shell()` instead.

Also, put some progress notices in the description, like:
```
currentBuild.description += "Built ${imageNames[i]}\n"
```